.github/workflows/upload_binary.yml

```diff
@@ -0,0 +1,57 @@
+name: Upload self-contained binaries
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
+        os: [ubuntu-16.04, windows-2019]
+        include:
+          - os: windows-2019
+            pathsep: ";"
+            executable_suffix: ".exe"
+            executable_mime: "application/vnd.microsoft.portable-executable"
+          - os: ubuntu-16.04
+            pathsep: ":"
+            executable_suffix: ".elf"
+            executable_mime: "application/x-executable"
+
+    steps:
+      - name: Get release asset upload URL from release tag
+        id: get_release
+        uses: bruceadams/get-release@8a90bac3fc6cb911f53d22ce10868b5831f2d244
```

Comment:
No Tag?

Reply:
Honestly I was scared that this PR would get rejected since it uses a third-party Action so I tried upping the changes this wouldn't get rejected by using a commit SHA instead of tags which are a weak point since tags can be modified. In hide-sight, creating non-signed binaries for other people to use is already sketchy so I don't know why I was freaking out over using a commit vs a tag :laughing:

Reply:
I haven't used `actions/upload-release-asset` myself so I'm not 100% sure how it works, but was wondering if it is possible to instead of
```
upload_url: ${{ steps.get_release.outputs.upload_url }}
```
use the gihub event context directly
```
upload_url: ${{ github.event.release.upload_url }}
```
thereby removing the need of the `bruceadams/get-release` action?

<sup>Documentation on [`github.event`](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context) context, and the [`release` payload](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#release).</sup>

Reply:
Oh thank you @anders-kiaer! I didn't know that the release event payload had such a attribute; didn't think of looking in the webhooks documentation. I'll definitely use your suggestion!